src/auth/tenant.ts

```diff
@@ -155,15 +192,30 @@ export class Tenant {
         allowPasswordSignup: false,
       });
     }
+    if (typeof response.mfaConfig !== 'undefined') {
+      this.multiFactorConfig = new MultiFactorAuthConfig(response.mfaConfig);
+    }
+    if (typeof response.testPhoneNumbers !== 'undefined') {
+      this.testPhoneNumbers = deepCopy(response.testPhoneNumbers || {});
+    }
   }
 
   /** @return {object} The plain object representation of the tenant. */
   public toJSON(): object {
-    return {
+    const json = {
       tenantId: this.tenantId,
       displayName: this.displayName,
       emailSignInConfig: this.emailSignInConfig && this.emailSignInConfig.toJSON(),
+      multiFactorConfig: this.multiFactorConfig && this.multiFactorConfig.toJSON(),
```

Comment:
With new versions of TS we can do this now:

```
multiFactorConfig: this.multiFactorConfig?.toJSON(),
```

Reply:
Done